src/pages/_app.tsx

```diff
@@ -26,21 +52,23 @@ const init = async () => {
  */
 const EuiApp: React.FC<AppProps> = ({ Component, pageProps }) => {
   React.useEffect(() => {
-    init();
+    init(window.location.pathname);
```

Comment:
see comments above but we shouldnt need to check pathname here

Reply:
how are we supposed to know which cookbook to look for in the cookbooks array without knowing the id from path/router?

Reply:
aye, we do it in the cookbook route, but that doesn't help if you reload the application while on a section, cause the :cookbook page is never loaded, if you then click a new section the cookbook ID it needs to find the section is never loaded

(copied here cause it seemed to not reply in my review)